### ISSUE TITLE
IPV6_DEFAULTGW is only set when ipv6 is set to dhcp

### ISSUE THREAD

**Issue by galexrt** [rhinstaller/anaconda#948]
I'm using Kickstart to configure static IPv4 + IPv6 on a server.
After the installation with Kickstart finished I noticed that IPv6 wasn't working due to the `IPV6_DEFAULTGW` not set in the network-script of the interface.

```
... --ipv6=_VALID_IPV6_HERE_::/64 --ipv6gateway=fe80::1 ...
```
It seems that the option `--ipv6gateway=...` is just used when the `ipv6` option is set to `dhcp`.
But in my case I need the option `IPV6_DEFAULTGW` to be used always when `--ipv6gateway=...` option is given.

Is this the wanted behavior to ignore the `ipv6gateway` option when `ipv6` option is not set to `dhcp` of Kickstart/Anaconda?

The current workaround is to manually add the `IPV6_DEFAULTGW` option to the network-scripts by using kickstart `%post` section.

***

See [dracut/parse-kickstart File Lines 453-462](https://github.com/rhinstaller/anaconda/blob/648c214bcfec25a1b806d4d649606f228fd572c7/dracut/parse-kickstart#L453-L462).

**Comment by rvykydal** [comment]
Thank you for spotting this.

We need to backport the fix from rhel7-branch, PR:
https://github.com/rhinstaller/anaconda/pull/950

**Comment by galexrt** [comment]
#950 is merged. Should I close this issue already or is there anything else that has to be done?

**Comment by galexrt** [comment]
Fixed by #950.
